lib/rules/block-closing-brace-newline-after/index.js

```diff
@@ -23,6 +23,7 @@ const messages = ruleMessages(ruleName, {
 const meta = {
 	url: 'https://stylelint.io/user-guide/rules/block-closing-brace-newline-after',
 	fixable: true,
+	deprecated: true,
```

Comment:
I wanted to check whether all the rules that were being deprecated were properly covered by prettier.

This one has [options](https://github.com/stylelint/stylelint/blob/main/lib/rules/block-closing-brace-newline-after/README.md#options):
- "always"
- "always-single-line"
- "never-single-line"
- "always-multi-line"
- "never-multi-line"

Are all of these properly covered by prettier?
If not, we need to make a list of rules that are being deprecated but are only partially covered by the prettier replacement.
e.g.

name | support | note
--- | --- | ---
rule A | full |
rule B | partial | does not support option X and Y
rule C | partial | only support option Z

Reply:
@Mouvedia was please suggested list created? To see which deprecated rule can be covered by the prettier replacement? Thanks.

Reply:
@jeddy3 @ybiquitous î

Reply:
Ideally, we might make a list, but we need more resources. Also, it may not be precious since the rules will be removed in the next major update.

However, we welcome your contribution if you have time.

Reply:
Following on from https://github.com/stylelint/stylelint/issues/6657#issuecomment-1427969428, I suggest the people working on the plugin pack draw up the list.

As @ybiquitous said, we've other things to focus on (that will benefit the majority of users). I suspect most people are already use a pretty printer alongside Stylelint, and most of those who don't will soon as it's a better solution.

Anyone wanting to continue using Stylelint for formatting pool work can together on a community plugin pack. As with custom syntaxes, the health of the plugin will be determined by the health of the community that maintains it.